### ISSUE TITLE
Extra Space in Submit Button Text

### ISSUE THREAD

**Issue by cherrykoda** [verbb/formie#845]
**Description**
The submit button text is getting an extra space on it that is not present in the field (it is therefore being introduced in the code). This is on a site that is not overwriting any formie templating.

<img width="484" alt="image" src="https://user-images.githubusercontent.com/3789834/165109267-66768908-8915-4cce-9532-56e767653a08.png">

<img width="963" alt="image" src="https://user-images.githubusercontent.com/3789834/165109570-6adddb9a-2920-4e55-911e-116e449324d9.png">

**Steps to reproduce**
1. Create a form, and add to a template
2. Inspect the submissions button

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Default
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.15
- Craft version: Craft Pro 3.7.38
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.16](https://github.com/verbb/formie/releases/tag/1.5.16)
